python/lsst/sdm_schemas/schemas/dp02_dc2.yaml

```diff
@@ -103,4890 +135,6840 @@ tables:
     datatype: boolean
     description: The Gaussianized PSF is bigger than the aperture. Forced on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1196
   - name: u_gaapPsfFlux
     "@id": "#Object.u_gaapPsfFlux"
     datatype: double
     description: GaaP flux with PSF aperture after multiplying the seeing by 1.15. Forced on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1197
   - name: u_gaapPsfFluxErr
     "@id": "#Object.u_gaapPsfFluxErr"
     datatype: double
     description: GaaP flux uncertainty with PSF aperture after multiplying the seeing by 1.15. Forced on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1198
   - name: u_hsmShapeRegauss_e1
     "@id": "#Object.u_hsmShapeRegauss_e1"
     datatype: double
     description: PSF-corrected shear using Hirata & Seljak (2003) regaussianization. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1199
   - name: u_hsmShapeRegauss_e2
     "@id": "#Object.u_hsmShapeRegauss_e2"
     datatype: double
     description: PSF-corrected shear using Hirata & Seljak (2003) regaussianization. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1200
   - name: u_hsmShapeRegauss_flag
     "@id": "#Object.u_hsmShapeRegauss_flag"
     datatype: boolean
     description: General failure flag, set if anything went wrong. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1201
   - name: u_hsmShapeRegauss_sigma
     "@id": "#Object.u_hsmShapeRegauss_sigma"
     datatype: double
     description: PSF-corrected shear using Hirata & Seljak (2003) regaussianization. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1202
   - name: u_iDebiasedPSF_flag
     "@id": "#Object.u_iDebiasedPSF_flag"
     datatype: boolean
     description: General failure flag, set if anything went wrong. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1203
   - name: u_iPSF_flag
     "@id": "#Object.u_iPSF_flag"
     datatype: boolean
     description: General failure flag, set if anything went wrong. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1204
   - name: u_iRound_flag
     "@id": "#Object.u_iRound_flag"
     datatype: double
     description: HSM moments. Measured on u-band.
     fits:tunit: pixel**2
+    tap:principal: 0
+    tap:column_index: 1205
   - name: u_i_flag
     "@id": "#Object.u_i_flag"
     datatype: boolean
     description: General failure flag, set if anything went wrong. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1206
   - name: u_inputCount
     "@id": "#Object.u_inputCount"
     datatype: int
     description: Number of images contributing at center, not including any clipping. Forced on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1207
```

Comment:
I think it'd make to put `inputCount` up by `footprintArea` by the general flags.